wherehows-web/app/styles/components/dataset-schema/_all.scss

```diff
@@ -1 +1,7 @@
 @import 'schema-table';
+
+.schemas-modified-time {
```

Comment:
`_all` partials should consist only of imports. you can create a new sibling `_dataset-schema.scss` file for this rule

Reply:
Ah didn't realize that's what _all meant. Will make change!